service/javadsl/integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -209,7 +209,7 @@ public void addStopHook(Callable<? extends CompletionStage<?>> hook) {
         ServiceInfo serviceInfo = ServiceInfo.of(serviceName);
 
         // ServiceClientLoader
-        CircuitBreakers circuitBreakers = new CircuitBreakers(actorSystem, new CircuitBreakerConfig(configuration),
+        CircuitBreakers circuitBreakers = new CircuitBreakersImpl(actorSystem, new CircuitBreakerConfig(configuration),
```

Comment:
Users who implemented their own Service locator and depended on CircuitBreakers. See for example [conductr-lib](https://github.com/typesafehub/conductr-lib/blob/master/lagom1-scala-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/lagom/scaladsl/ConductRApplicationComponents.scala#L45).

IIUC, there is no longer a `CircuitBreakers` constructor.